File: base/src/main/java/com/zhangteng/base/mvp/base/BaseLoadingView.java
package com.zhangteng.base.mvp.base;

/**
 * Created by swing on 2019/7/16 0016.
 */
public interface BaseLoadingView<T> extends BaseView {
    void showLoadingView();

    void dismissLoadingView();

    void inflateView(T data);
}

File: base/src/main/java/com/zhangteng/base/mvp/base/BaseLoadingView.kt
package com.zhangteng.base.mvp.base

/**
 * Created by swing on 2019/7/16 0016.
 */
interface BaseLoadingView<T> : IView {
    open fun showLoadingView()
    open fun dismissLoadingView()
    open fun inflateView(data: T?)
}